src/main/java/mate/academy/springboot/datajpa/model/Category.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.springboot.datajpa.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+
+@Entity
+@Table(name = "categories")
+@Data
```

Comment:
It is not recomended to use @Data with entities. Something connected with toString & LazyInit Exception